### ISSUE TITLE
Unable to install in fresh project due to unsatisfiable requirements on ReversePropagation

### ISSUE THREAD

**Issue by simonp0420** [dpsanders/SimpleSATSolver.jl#8]
I watched your talk on this today and it looked like exactly what I need for my current project.  But I'm having a problem installing it.  Any advice would be appreciated:

```
julia> mkdir("tt")                                                                                                 
"tt"                                                                                                               
                                                                                                                   
julia> cd("tt")                                                                                                    
                                                                                                                   
(@v1.6) pkg> activate .                                                                                            
  Activating new environment at `D:\peter\Documents\consulting\LBFP\tt\Project.toml`                               
                                                                                                                   
(tt) pkg> status                                                                                                   
      Status `D:\peter\Documents\consulting\LBFP\tt\Project.toml` (empty project)                                  
                                                                                                                   
(tt) pkg> add https://github.com/dpsanders/SatisfiabilityInterface.jl                                              
    Updating git-repo `https://github.com/dpsanders/SatisfiabilityInterface.jl`                                    
    Updating registry at `C:\Users\peter\.julia\registries\General`                                                
    Updating git-repo `https://github.com/JuliaRegistries/General.git`                                             
   Resolving package versions...                                                                                   
ERROR: Unsatisfiable requirements detected for package ReversePropagation [527681c1]:                              
 ReversePropagation [527681c1] log:                                                                                
 ├─possible versions are: 0.1.0-0.1.1 or uninstalled                                                               
 ├─restricted to versions 0.1 by SatisfiabilityInterface [f4c7ffe1], leaving only versions 0.1.0-0.1.1             
 │ └─SatisfiabilityInterface [f4c7ffe1] log:                                                                       
 │   ├─possible versions are: 0.1.0 or uninstalled                                                                 
 │   └─SatisfiabilityInterface [f4c7ffe1] is fixed to version 0.1.0                                                
 └─restricted by compatibility requirements with Symbolics [0c5d862f] to versions: uninstalled — no versions left  
   └─Symbolics [0c5d862f] log:                                                                                     
     ├─possible versions are: 0.1.0-1.4.2 or uninstalled                                                           
     └─restricted to versions 1 by SatisfiabilityInterface [f4c7ffe1], leaving only versions 1.0.0-1.4.2           
       └─SatisfiabilityInterface [f4c7ffe1] log: see above
```

**Comment by dpsanders** [comment]
Oops, sorry about that. One option is to do

] dev ReversePropagation

And update the dependencies there. Otherwise I'll get to this later or tomorrow.

**Comment by simonp0420** [comment]
Sorry, but I don't understand how a modification to ReversePropagation would fix this.  I don't really understand this line above:
```restricted by compatibility requirements with Symbolics [0c5d862f] to versions: uninstalled — no versions left```
How can Symbolics require ReversePropagation to be uninstalled?

**Comment by dpsanders** [comment]
`ReversePropagation` was restricting the version of `Symbolics` to an older version.

I have just updated this in `ReversePropagation` so please update in half an hour (?) to get version 0.1.2 of ReversePropagation, which should hopefully solve this problem. Please let me know if there's still an issue.

**Comment by simonp0420** [comment]
Thanks very much for doing this so quickly during a busy time (Juliacon).  This may be old news to you, but it took me a while to figure out that after your fix, I was still getting the same error until I did a `Pkg.update()`.  Apparently when adding an unregistered package using a URL, the package manager doesn't update the local registry cache, so it was still trying to grab the old version of `ReversePropagation` until I forced the update.  But it's all working now.  Again, thanks!
